dockers/docker-orchagent/start.sh

```diff
@@ -12,6 +12,17 @@ function start_app {
     done
 }
 
+function config_acl {
+    if [ -f "/etc/sonic/acl.json" ]; then
+        mkdir -p /etc/swss/config.d/acl
+        rm -rf /etc/swss/config.d/acl/*
```

Comment:
i would prefer to have the acl configuration part to be there in config.sh script. we could have the intermediate acl json files stored here. it doesn't hurt and could be used as references.

as a separate suggestion that we could do it later, i think we could just put the configurations that we need into the config.d folder, copying them from /usr/share/sonic/... folder. in sonic-swss repository, we only have samples there.